google/cloud/aiplatform/__init__.py

```diff
@@ -32,6 +32,7 @@
     CustomContainerTrainingJob,
```

Comment:
Are the changes in this file needed?

Reply:
Yes, it allows the user to directly access the class via `aiplatform.AutoMLForecastingTrainingJob` instead of `aiplatform.training_jobs.AutoMLForecastingTrainingJob`